docs/essentials/configuring-pre-flight-checks.md

```diff
@@ -0,0 +1,130 @@
+---
+Description: This guide shows you how to set up pre-flight checks run before each pipeline in Semaphore 2.0.
+---
+
+!!! warning "This feature is in the private beta stage and a part of the Semaphore Enterprise feature plan."
```

Comment:
Let's add here `If you want to use it contact us...` line similar to the one [here](https://docs.semaphoreci.com/ci-cd-environment/self-hosted-agent-types/)